Yoast/wordpress-seo
# Always hook the notification check

Should show notice returns false when the option is disabled, removing the notification.

## Summary

This PR can be summarized in the following changelog entry:

* Fixes a bug where the Ryte notification is not removed when disabling the Ryte feature.

## Relevant technical choices:

* Moved the notification hook outside of the `activated` check.

## Test instructions

This PR can be tested by following these steps:

* Don't check the `Discourage search engines from indexing this site` in your `Reading` settings in WordPress
* Activate the Ryte check on the Yoast SEO Dashboard widget
* See the notification being added about the frontpage not being able to index
* Disable the Ryte feature and see the notification being removed with this PR

## Quality assurance


Fixes #9093
